### DISCUSSION TITLE
lambda calculus 內容方向及大綱

### DISCUSSION THREAD

**Discussion by FizzyElt** [FizzyElt/functional-programming#104] [General]
我對這部份還不算太了解，目前朝向把 lambda calculus 基本概念帶過，以及實務上會聚焦在哪部份

1. 基本概念 : 可能是蠻巨大的主題，先以入門內容優先
2. 實務上我們會聚焦在哪部份 :

**Comment by dannypsnl** [comment]
入門應該就是 SML 的部分？像是 OCaml 的子集合這樣
應該是先熟悉 let 跟 lambda
然後知道 operational semantic
再來觀察 lambda calculus，證明 computation 能力

**Reply by FizzyElt** [reply]
SML 全名是什麼
還有 let 跟 lambda 的關係是 ?

**Reply by dannypsnl** [reply]
1. Standard Meta Language
2. let 是 OCaml 的基本定義方式，lambda 就是 lambda。在 scheme 系慣例是把 let 攤開成 lambda，但有型別的時候這個作法不是通常可行
